integration/tests/common/ghost_node_example_test.go

```diff
@@ -41,9 +39,11 @@ func TestGhostNodeExample_Subscribe(t *testing.T) {
 
 		// a verification node
 		verNode = testnet.NewNodeConfig(flow.RoleVerification, testnet.WithLogLevel(zerolog.FatalLevel))
+
+		accessNode = testnet.NewNodeConfig(flow.RoleAccess, testnet.WithLogLevel(zerolog.FatalLevel))
 	)
 
-	nodes := append([]testnet.NodeConfig{collNode, conNode1, conNode2, conNode3, realExeNode, verNode, ghostExeNode})
+	nodes := append([]testnet.NodeConfig{collNode, conNode1, conNode2, conNode3, realExeNode, verNode, ghostExeNode, accessNode})
 	conf := testnet.NewNetworkConfig("ghost_example_subscribe", nodes)
 
 	net := testnet.PrepareFlowNetwork(t, conf)
```

Comment:
This test needs to call `testnet.PrepareFlowNetwork(t, conf)` to setup all the docker instances. That call need the `*testing.T` object and that is not available if I convert this test to an `ExampleTest` :(

Reply:
Ahhh I see :/

Okay, in that case it's okay, can we at least update https://github.com/onflow/flow-go/pull/1670/files#diff-6df27b5bc6af729e3267f2404aa0d5e17319d629fbfffd697f1272474f59cfd2L19-L20 to explain that these tests are actually meaningful tests, and don't solely exist to serve as examples? thanks :)